v4-client-js/src/clients/composite-client.ts

```diff
@@ -247,6 +255,73 @@ export class CompositeClient {
     return Math.round(future.getTime() / 1000);
   }
 
+  /**
+     * @description Place a short term order with human readable input.
+     *
+     * @param subaccount The subaccount to place the order on.
+     * @param params The parameters of the order to place.
+     *
+     * @throws UnexpectedClientError if a malformed response is returned with no GRPC error
+     * at any point.
+     * @returns The message to be passed into the protocol
+     */
+  async placeShortTermOrderMsgs(
+    subaccount: SubaccountInfo,
+    params: IHumanReadableShortTermOrder,
+  ): Promise<EncodeObject[]> {
+    const {
+      marketId,
+      side,
+      price,
+      size,
+      clientId,
+      goodTilBlock,
+      timeInForce,
+      reduceOnly,
+    } = params;
+
+    await this.validateGoodTilBlock(goodTilBlock);
+
+    const marketsResponse = await this.indexerClient.markets.getPerpetualMarkets(marketId);
```

Comment:
Can we add another parameter to `placeShortTermOrderMsgs` so that these Indexer `GET` requests can be optional?